lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/Command.scala

```diff
@@ -27,6 +28,23 @@ case class Command(command: Seq[String], extraEnv: Seq[(String, String)]) {
       process.waitFor()
     }
 
+  /** Runs the command and returns its exit code along with any output produced..
+    *
+    * May return an exception if it is impossible to run the command (for
+    * example due to insufficient permissions or nonexistent executable).
+    */
+  def runAndCaptureOutput(): Try[(Int, String)] =
+    wrapError {
+      logger.debug("Executing {}", this)
+      val processBuilder = builder()
+      val process        = processBuilder.start()
+      val wrappedProcess = new WrappedProcess(command, process)
+      val stringBuilder  = new StringBuilder()
+      wrappedProcess.appendToBuilder(stringBuilder)
```

Comment:
The method name `appendToBuilder` is misleading - it sounds like it adds 'what it has currently stored' to the builder and nothing more - which makes this code look incorrect.

But the code is correct, just the name suggests something else.

Could you please rename it? I'd suggest `registerStringBuilderAppendTarget` or `registerStringBuilderForFutureAppends` or something like that.